source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,26 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
+	unlock: override func
+}
+
+MutexGlobal: class extends Mutex {
+	_globalMutex := static Mutex new(MutexType Safe)
+	init: func
+	lock: override func {
+		This _globalMutex lock()
+	}
+	unlock: override func {
+		This _globalMutex unlock()
+	}
+	cleanup: static func {
```

Comment:
In `Closure` is should probably be `free`. At some point, we had an ambition to name cover destructors `dispose`. I don't remember why, though (and I'm also not sure why `Closure` is a cover).